ui/webui/src/components/review/ReviewConfiguration.jsx

```diff
@@ -96,6 +96,10 @@ const DeviceRow = ({ deviceData, disk, requests }) => {
         const partitionName = Object.keys(deviceData).find(device => deviceData[device].name.v === req["device-spec"]);
         const device = deviceData[partitionName];
 
+        if (!req.reformat && req["format-type"] !== "biosboot" && req["mount-point"] === "") {
```

Comment:
I decided to show the biosboot partition here mostly because we already check for its presence in the tests, we might want to hide it too, thoughts?

Reply:
I don't think because of the tests we should keep that. Is it anyhow useful info for the user?
I suggest not - so we can drop it.

Reply:
@garrett what do you say?

Reply:
Ok, removed.